### PR TITLE
Add WDS

### LINKED ISSUE #53: Add USWDS to foia
Pull in the [US Web Design Standards](https://www.npmjs.com/package/uswds) source from npm so that we can extend and control the styles with specific granularity.

**Acceptance criteria**
- [x] uswds is pulled in from npm so that we can keep up to date with new versions
- [x] Apply the uswds grid to the jekyll layout templates (no need to apply grid to existing foia.gov content)

### PR DESCRIPTION
**Pull request by hbillings**
Closes #53. This moves old legacy stuff into its own legacy templates, while creating new WDS-based templates. This allows us to move old stuff over gradually but also keep developing the new stuff.

### CODE REVIEW

File: www.foia.gov/foia-legacy.css [hunk 1]
```diff
@@ -8,7 +8,7 @@
 html {background: #121B26;}
 body {font: 14px Calibri, "Trebuchet MS", Arial, Helvetica, sans-serif normal; color: #333; }
 #background {background: url(images/bg-grad.jpg) repeat-x top left }
-#container {display: block; width: 960px; margin: 0 auto;}
+/*#container {display: block; width: 960px; margin: 0 auto;}*/
```
Comment (adborden) [inline www.foia.gov/foia-legacy.css]: I think these are OK to delete :)

File: www.foia.gov/_includes/footer.html [hunk 2]
```diff
@@ -1,25 +1,18 @@
...
-  </ul>
-  </div>
-  <div class="clear">&nbsp;</div>
-  <div align="center">
-    <a href="http://www.data.gov/"><img src="images/datagov-footer.gif" alt="data.gov" /><span>Data.gov</span></a>
-	<a href="http://www.whitehouse.gov/open"><img src="images/opengov-footer.gif" alt="whitehouse.gov/open" /><span>Whitehouse.gov/open</span></a>
-	<a href="http://www.usaspending.gov/"><img src="images/usaspending-footer.gif" alt="usaspending.gov" /><span>USASpending.gov</span></a>
-    <a href="http://www.recovery.gov/"><img src="images/recoverygov-footer.gif" alt="recovery.gov" /><span>Recovery.gov</span></a>
-  </div>
+{% unless page.not_legacy %}
```
Comment (adborden) [inline www.foia.gov/_includes/footer.html]: This is a little funny with the double negative... maybe we want to swap this to `page.modern` or `page.uswds`?

File: www.foia.gov/request.html [hunk 3]
```diff
@@ -2,5 +2,6 @@
 title: Create a request
 layout: default
 navigation: request
+not_legacy: true
```
Comment (adborden) [inline www.foia.gov/request.html]: How does someone know if the page should be legacy or not?

**Comment by adborden** [comment]
Looks good. I've got some questions, but feel free to merge away :anchor:

File: www.foia.gov/_includes/footer.html (hunk 2 above)
Comment (hbillings) [reply www.foia.gov/_includes/footer.html]: I like `page.uswds`! Will change.

File: www.foia.gov/request.html (hunk 3 above)
Comment (hbillings) [reply www.foia.gov/request.html]: Originally I had this as a `legacy` identifier in the frontmatter of pages we're keeping as-is, and I think it just morphed too far. Do you think changing it to `use_uswds` or similar makes it clearer?

**Pull request merged**